scroll-animations-1/Overview.bs

```diff
@@ -922,9 +922,10 @@ spec: cssom-view-1; type: dfn;
 	through those APIs.
 
 	If the final style and layout update
-	would result in a change of time for any [=scroll progress timelines=] or
-	[=view progress timelines=], they will not be re-sampled to reflect the
-	new offset until the next update of the rendering.
+	would result in a change the time or scope (see 'timeline-scope')
```

Comment:
nit: "would result in a change the time or scope" => "would change the time or scope"

?

Reply:
ah, yeah, that looks non-grammatical. :P Fixed (slightly differently). Thanks!